backend/src/main/scala/cromwell/backend/validation/ValidatedRuntimeAttributesBuilder.scala

```diff
@@ -47,25 +47,13 @@ trait ValidatedRuntimeAttributesBuilder {
 
   def unsupportedKeys(keys: Seq[String]): Seq[String] = keys.diff(validationKeys)
 
-  private lazy val coercionMap: Map[String, Traversable[WdlType]] =
-    validations.map(attr => attr.key -> attr.coercion).toMap
-
   private lazy val validationKeys = validations.map(_.key)
 
-  private lazy val staticDefaults: Map[String, WdlValue] = (validations collect {
-    case attr if attr.staticDefaultOption.isDefined => attr.key -> attr.staticDefaultOption.get
-  }).toMap
-
-  def build(attrs: Map[String, WdlValue], options: WorkflowOptions, logger: Logger): ValidatedRuntimeAttributes = {
-    import RuntimeAttributesDefault._
-
+  def build(attrs: Map[String, WdlValue], logger: Logger): ValidatedRuntimeAttributes = {
     // Fail now if some workflow options are specified but can't be parsed correctly
```

Comment:
I think we would never actually fail because we see an unrecognised runtime attribute?

(the validation picks the values it expects out of the list, ignoring everything else)